packages/v4/src/content/design-guidelines/components/popover/popover.md

```diff
@@ -2,50 +2,62 @@
 id: Popover
 section: components
 related: [
-  'Button',
-  'Content',
-  'Tooltip'
+ 'Button',
+ 'Content',
+ 'Tooltip'
 ]
 ---
-
-A **popover** is in-app messaging that provides more information on specific product areas. Popovers display content in a new window that overlays the current page. Unlike modals, popovers don’t block the current page. 
-
+ 
+A **popover** is in-app messaging that provides more information on specific product areas. Popovers display content in a new window that overlays the current page. Unlike modals, popovers don’t block the current page.
+ 
 Popovers are triggered by a click and dismissed in one of the following ways:
-- The user clicks the exit icon. 
+- The user clicks the exit icon.
 - The user clicks anywhere on the screen outside the popover.
 - The user clicks an action button inside the popover. (Note: This won't always close the popover; it depends on the action button.)
-
+ 
 <img src="./img/popover-example.png" alt="popover example with placeholder text for the title and informational text" width="300"/>
-
+ 
 For information on other forms of on-screen help, see [Tooltip](/components/tooltip/design-guidelines) and [Hint](/components/hint/design-guidelines).
-
+ 
+## **Popover types**
```

Comment:
No need for this to be bolded since it's a header, so just gotta remove the ** !